src/orchestrator-tracing/src/lib.rs

```diff
@@ -129,6 +129,17 @@ pub struct TracingCliArgs {
         use_value_delimiter = true
     )]
     pub opentelemetry_resource: Vec<KeyValueArg<String, String>>,
+    /// Default the OpenTelemetry tracing collector to off, but allow it to be
+    /// dynamically turned on.
+    ///
+    /// Requires that the `--opentelemetry-endpoint` option is specified.
+    #[clap(
+        long,
+        env = "OPENTELEMETRTY_ENABLED",
+        default_value_t = DefaultTrue::default(),
```

Comment:
I'm confused, why is this not a boolean, and why isn't this `default_value_t = true`?

Reply:
clap doesn't support `bool` with default true :(

Reply:
How about we flip it to `--opentelemetry-disabled`, then we can pass that on cloud; default will still be correct but we save ourselves some weird cli logic

Reply:
@antifuchs pr is updated with this change!